pkg/registries/artifactory/artifactory.go

```diff
@@ -7,16 +7,22 @@ import (
 )
 
 // Creator provides the type and registries.Creator to add to the registry of image registries.
-func Creator() (string, func(integration *storage.ImageIntegration) (types.Registry, error)) {
-	return "artifactory", func(integration *storage.ImageIntegration) (types.Registry, error) {
-		return docker.NewDockerRegistry(integration, false)
-	}
+func Creator() (string,
+	func(integration *storage.ImageIntegration, _ *types.CreatorOptions) (types.Registry, error),
```

Comment:
If we touch this anyways, it'd be nice to make this function an alias in `types`.